website/docs/docs/cloud/git/import-a-project-by-git-url.md

```diff
@@ -102,7 +102,20 @@ CodeCommit uses `master` as its default branch, and to initialize your project,
 - Select **Settings** -> **Edit** and under **General Settings**, check the **Default to a custom branch** checkbox.
 - Specify the custom branch as `master` and click **Save**.
 
-🎉 **You're all set!** Once dbt Support handles your request and you've set your custom branch, your project is ready to execute dbt runs on dbt Cloud. 
+#### Step 5: Configure pull request (PR) template URLs (Optional)
+
+To prevent users directly merging code changes into the default branch, configure the PR Template URL in the **Repository details** page for your project. Once configured, dbt Cloud will prompt users to open a new PR after committing and synching code changes on the branch in the IDE, before merging any changes into the default branch.
```

Comment:
```suggestion
To prevent users from directly merging code changes into the default branch, configure the [PR Template URL](/docs/collaborate/git/pr-template) in the **Repository details** page for your project. Once configured, dbt Cloud will prompt users to open a new PR after committing and synching code changes on the branch in the IDE, before merging any changes into the default branch.
```